Add Beta Ceph metrics module to Kibana Home

ruflin opened the pull request:
Add beta `Ceph` module to Metrics modules.

<img width="2043" alt="screen shot 2018-06-29 at 12 43 22 pm" src="https://user-images.githubusercontent.com/373691/42109208-1aedc92e-7b9a-11e8-8c96-1c9be12a10d1.png">

<img width="2042" alt="screen shot 2018-06-29 at 12 43 30 pm" src="https://user-images.githubusercontent.com/373691/42109217-1f81ce40-7b9a-11e8-8eca-f3e8695b9e7f.png">

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic-kibana-pull-request/5558/)

nreese commented:
@gchaps Do you want to review?

gchaps commented:
Read through the text. LGTM

Minor: Until the icon is ready, you could shift the text in the Ceph metrics box to the left on the Add Data page.

ruflin commented:
@gchaps Could you share on how I can do that?

cchaos commented:
@ruflin I believe if you just comment out this line: https://github.com/ruflin/kibana/blob/3976b57ea5cd762dccc69edeabc99a023ecb9053/src/core_plugins/kibana/server/tutorials/ceph_metrics/index.js#L34

It should layout that card without an icon since that icon does not actually exist yet.

ruflin commented:
@cchaos Thanks for your help. Change applied.

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic-kibana-pull-request/6147/)

ruflin commented on src/core_plugins/kibana/server/tutorials/ceph_metrics/index.js:
```diff
@@ -0,0 +1,48 @@
+    artifacts: {
+      application: {
+        label: 'Discover',
+        path: '/app/kibana#/discover'
```
@tsg @alexfrancoeur I'm jumping here now to the discovery page. I was thinking if we should jump to discovery and already filter for the data from the module?

tsg replied:
Another option is to leave that out and then there is simply no button at the end. But I think Discover is not bad.

ruflin replied:
I tried this at first but it feels awkward as the user kind of gets stuck. Everything is complete, what now? Will move forward with discover for now and can still improve it in the future.

ruflin commented:
I rebased this PR now on top of https://github.com/elastic/kibana/pull/20514 to get the new check data feature and added a link to discovery. Please have a look again.

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic-kibana-pull-request/6844/)

ruflin commented:
@tsg Thanks a lot for your changes in #20514. Makes adding new modules even easier.
